google/earthenterprise
# Allow extracting globe packets from portable file

Addresses #1072 

When a 3d cut file is provided as input to `geglxinfo` the packets are dumped in a manner similar to 2d packet dumps.  Vector data isn't quite human readable but gives a hint at the contents of the packet.

Verification steps:
* compile or obtain `geglxinfo` for this PR as well as `geglxinfo` from master
* obtain or create the following files:
  1. 2d `.glm` file
  2. `.glc` file with 2d contents
  3. `.glc` file with 3d contents
  4. 3d `.glb` file
* Run original `geglxinfo --glx <filename> --extract_packets --layer_idx 0 &> output.txt` against file types 1 and 2 above and save the output from each
* Run PR `geglxinfo --glx <filename> --extract_packets --layer_idx 0 &> output.txt` against all portable file types and save the output
* For file types 1 and 2, ensure output of new `geglxinfo` matches the output of the old `geglxinfo` exactly.
* Ensure output of new `geglxinfo` for file types 3 and 4 produces a `maptiles` directory with all the different expected subfiles and data
